guides/v2.1/architecture/archi_perspectives/framework.md

```diff
@@ -54,120 +56,29 @@ The `vendor/magento/framework` directory maps to the `Magento\Framework` {% glos
 
 The Magento Framework (`lib/internal/Magento/Framework/`) provides a robust range of functionality. If you are an {% glossarytooltip 55774db9-bf9d-40f3-83db-b10cc5ae3b68 %}extension{% endglossarytooltip %} developer, you may be interested in this subset of Framework namespaces.
 
-<table>
-   <tbody>
-      <tr style="background-color: lightgray">
-         <th>Namespace</th>
-         <th>Purpose</th>
-      </tr>
-      <tr>
-         <td><code>Magento\Framework\DataObject</code>
-         </td>
-         <td>Provides standard functionality for storing and retrieving data through magic methods. This is the base class for many Magento classes.</td>
-      </tr><tr>
-         <td><code>Magento\Framework\Model</code>
-         </td>
-         <td>Contains base Model classes that almost all Magento Model classes extend from.</td>
-      </tr><tr>
-         <td><code>Magento\Framework\Model\AbstractModel</code>
-         </td>
-         <td />
-      </tr>
-      <tr>
-         <td><code>Magento\Framework\Model\ResourceModel\AbstractResource</code></td>
-         <td />
-      </tr>
-      <tr>
-         <td><code>Magento\Framework\Controller</code></td>
-         <td>Contains classes to help return different types of results (for example, JSON and redirects).</td>
-      </tr>
-      <tr>
-         <td><code>Magento\Framework\View</code></td>
-         <td>Contains code to render pages and layouts.</td>
-      </tr><tr>
-         <td><code>Magento\Framework\Data</code></td>
-         <td>Contains additional classes that handle forms.</td>
-      </tr><tr>
-         <td><code>Magento\Framework\Url</code></td>
-         <td>Contains code to look up other pages in Magento.</td>
-      </tr>
-   </tbody>
-</table>
+|Namespace|Purpose|
```

Comment:
wrap each namespace in back-ticks like `Magento\Framework\DataObject`

Reply:
Done.

Reply:
I don't see the changes